sympy/stats/stochastic_process_types.py

```diff
@@ -842,13 +842,31 @@ def _transient2absorbing(self):
         return ImmutableMatrix(t2a)
 
     def fundamental_matrix(self):
+        """
+        Each entry fundamental matrix can be interpreted as
+        the expected number of times the chains is in state j
+        if it started in state i.
```

Comment:
This definition makes it seem like your matrix should just contain `oo` in all its entries. Every state is visited an infinite number of times in an ergodic chain.

Reply:
I'll check for the more clear definition.